### ISSUE TITLE
A different result

### ISSUE THREAD

**Issue by huhengtong** [Coolnesss/fada-pytorch#1]
I run the main.py and just got a ~40% accuracy, I have not figured out the reason. Anybody else has encountered this problem?

**Comment by Coolnesss** [comment]
Hey,
That's probably because I didn't set a random seed. The method doesn't appear to be too robust to changes in initialization. Run it again to try with a different seed.

**Comment by huhengtong** [comment]
Hey,
I found a problem about your data.py. They used all training samples  of the source domain (MNIST) for training in the paper. However,  you just sample a subset (n=2000) for training. Do you have a reason for this? 
Thanks for your reply.

**Comment by Coolnesss** [comment]
Have a look at Section 4.1, where they say 

> which involves randomly selecting 2000 images from MNIST and 1800 images from USPS.

My interpretation is that 2000 training images are used.

**Comment by huhengtong** [comment]
For task M->U, U->M they sampled 2000, but for task M->S (which is your experiment) they used all training samples. The section 4.1 truly has such a introduction.

**Comment by Coolnesss** [comment]
Yeah you seem to be correct, I must have misunderstood the paper. Feel free to drop a PR to correct it ;)

**Comment by huhengtong** [comment]
Hey,
I try to use the USPS dataset to implement the experiment. However, I got a very bad result(around 0.12), I'm now trying to fix the code, do you have some suggestions?
